Condense PR template (#9574)
endless-sky/endless-sky: .github/pull_request_template.md

> (The lines in brackets (like this one) is instructions for filling out this template. These lines can be deleted.)

Reviewer: ```suggestion
(The lines in brackets (like this one) are instructions for filling out this template. These lines can be deleted.)
```

Reviewer 2: ```suggestion
(The lines in parentheses (like this one) are instructions for filling out this template. These lines can be deleted.)
```

(The lines in double curly brackets ({{}}) should be replaced as it describes.)
(You can open a PR to add or improve this template, if you find it lacking!)

(Choose one heading, or add your own.)
**Content (Artwork / Missions / Jobs)**
**Balance**
**Feature**
**Bug fix**
**CI/CD/Testing**
**Documentation**

This PR addresses the bug/feature described in issue #{{insert number}}

## Summary
{{Describe and explain your changes. Include links to related issues.}}

## Screenshots
{{Include before and after screenshots demonstrating your changes, if applicable.}}

## Usage examples
{{If your changes affect how game data can be defined, provide examples demonstrating the changes here.}}

## Testing Done

> {{Describe how you tested these changes. How they meet the purpose. Ensure they don't introduce new issues.}}

Reviewer: What does "how they meet the purpose" mean? Is it in relation to the issue this PR addresses or whether the test itself works?

Reviewer 2: ```suggestion
{{Describe how you tested these changes. Ensure that new issues aren't introduced.}}
```

{{If this is a new feature, have you added any automated tests using the unit or integration testing framework?}}

## Save File
This save file can be used to test these changes:
{{Attach a save file that allows people to easily test your added mission content or see your new in-game art.}}

> ## Artwork Checklist

Reviewer 2: ```suggestion
## Artwork Checklist
{{If any artwork was added or changed by this PR, the following must be provided.}}
```
